sherpa/ui/tests/test_session.py

```diff
@@ -21,6 +21,13 @@
 from numpy.testing import assert_array_equal
 
 
+# bug #303
+def test_set_log():
```

Comment:
Can you also add

```
def test_set_linear():
    session = Session()
    session.set_xlinear()
    session.set_ylinear()
```

(or add the set_x/ylinear calls to `test_set_log`?) I don't expect there to be a problem there, but these functions are directly related to the set_x/ylog calls, so would be good to test both).

I didn't want to add to the PR directly in case you were working on it for my previous comment.